docs/policygenerator-reference.yaml

```diff
@@ -137,10 +138,10 @@ policies:
         metadataComplianceType: ""
          # Optional. (See policyDefaults.evaluationInterval for description.)
         evaluationInterval: {}
-        # Optional. A Kustomize patch to apply to the manifest(s) at the path. If there
-        # are multiple manifests, the patch requires the apiVersion, kind, metadata.name,
-        # and metadata.namespace (if applicable) fields to be set so Kustomize
-        # can identify the manifest the patch applies to.
+        # (deprecated) Optional. A Kustomize patch to apply to the manifest(s) at the path. If 
```

Comment:
I also figured in-config patches would be deprecated with this addition.

Reply:
I definitely think using another kustomization.yaml to apply the patches is a better solution, so I think we should encourage that somehow. To me, saying it's "deprecated" implies we plan on removing it - so do we? What would the timeline be?

If we don't plan on actually removing the option, I think we could just suggest that using another kustomization.yaml file is a better solution.

Reply:
I've added a note to the deprecation to use a Kustomize directory instead.